### PR TITLE
RedpandaStats

### PR DESCRIPTION
**Pull request by hythloda**

### CODE REVIEW

File: RedpandaStats/README.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+# RedPanda and Deephaven
```
Comment (margaretkennedy) [inline RedpandaStats/README.md:1]: general question - do we want to develop this into a longer article, posted on deephaven.io under Examples, and keep what is in examples just code and data? Or leave this as is to support an Example article / blog post

File: RedpandaStats/kafka-produce.py [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+#
+# == Example
+#
+# From [http://localhost:10000/ide](http://localhost:10000/ide/):
+# from deephaven import KafkaTools as kt
+# from deephaven import Types as dht
+# result= kt.consumeToTable({'bootstrap.servers': 'redpanda:29092'} , 'docker-stats', key=kt.IGNORE, value=kt.json([
+#    ('container', dht.string),
+#    ('name',   dht.string),
+#    ('cpu',  dht.string),
```
Comment (jcferretti) [inline RedpandaStats/kafka-produce.py:31]: Do we need these columns to have a more meaningful types than `string` to be able to do interesting things with them? (eg average, sums across containers, whatever)

Comment (hythloda) [reply RedpandaStats/kafka-produce.py:31]: I changed the ones that make sense to change. There is logic to change others (memoryUsage, memoryLimit, networkInput, networkOutput, blockInput, blockOutput), but then we have unit conversion to worry (ie B vs kB vs MB and GB and GiB vs MiB)

File: RedpandaStats/README.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+### To run script
+
+
+In your 'RedpandaStats' directory bring up this version of the deployment:
+
+```bash
+docker-compose up -d
+```
+
+This starts the needed containers needed for RedPanda and Deephaven.
```
Comment (margaretkennedy) [inline RedpandaStats/README.md:34]:
```suggestion
This starts the containers needed for RedPanda and Deephaven.
```

File: RedpandaStats/README.md [hunk 4]
```diff
@@ -0,0 +1,60 @@
...
+
+To start listening to the Kafka topic docker-stats, navigate to [http://localhost:10000/ide](http://localhost:10000/ide/) and enter:
+
+```python
+from deephaven import KafkaTools as kt
+from deephaven import Types as dht
```
Comment (margaretkennedy) [inline RedpandaStats/README.md:40]:
```suggestion
from deephaven import Types as dht

```

File: RedpandaStats/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
...
+    ('memoryPercent',   dht.double),
+    ('networkInput',  dht.string),
+    ('networkOutput',    dht.string),
+    ('blockInput',  dht.string),
+    ('blockOutput',    dht.string),
+    ('pids',    dht.int32)
+    ]),table_type = 'append')
+  ```
+
+  In your terminal to produce the kafka stream execute the kafka-produce.py script:
```
Comment (margaretkennedy) [inline RedpandaStats/README.md:56]:
```suggestion
  To produce the Kafka stream, execute the `kafka-produce.py` script in your terminal:
```

**Review by margaretkennedy — APPROVED** [review]

**Pull request merged**